### PR TITLE
Aggregate Opportunity

### PR DESCRIPTION
**Pull request by HalimaSalifu**
I updated what I had forked to my repository in order to identify possible ways to aggregate methods. I have done so in order to show the similar methods and place together.

### CODE REVIEW

File: doctoshotgun.py [hunk 1]
```diff
@@ -347,7 +331,10 @@ def normalize(cls, string):
             [c for c in nfkd if not unicodedata.combining(c)])
         normalized = re.sub(r'\W', '-', normalized)
         return normalized.lower()
-
+    
+    
+    class BookAppointment(JSonPage):
+    ''' Place all instances of booking an appointment in one place '''
```
Comment (gitolicious) [inline doctoshotgun.py:337]: Compilation fails at this line, please double-check
> IndentationError: expected an indented block

File: doctoshotgun.py [hunk 2]
```diff
@@ -347,7 +331,10 @@ def normalize(cls, string):
             [c for c in nfkd if not unicodedata.combining(c)])
         normalized = re.sub(r'\W', '-', normalized)
         return normalized.lower()
-
+
```
Comment (gitolicious) [inline doctoshotgun.py:334]: Please run [Prettier](https://prettier.io/) before committing so we keep a consistent code format.

**Review by gitolicious — CHANGES_REQUESTED** [review]
You indentation looks weird as you don't indent the content of classes. Also there are quire a few empty lines which Prettier will help you get rid off.

**Comment by rbignon** [comment]
Hello,

I appreciate your contribution, but please tell your teacher to ask you to contribute to opensource softwares with useful features, not by applying a design pattern with your brain turned off.

Thank you.

**Pull request closed without merging**